test/common/common/utility_speed_test.cc

```diff
@@ -7,6 +7,8 @@
 #include "absl/strings/string_view.h"
 #include "testing/base/public/benchmark.h"
 
+using namespace Envoy;
```

Comment:
Can this code be put into namespace Envoy, instead of adding a "using"?

Reply:
I see two options: 
- move test code into Envoy namespace (most test code is already in that namespace - only few tests run in default ns)
- change MACRO to use FQ namespace Envoy::Logger::Logger instead of Logger::Logger.

I think that first option is a bit cleaner - will try to do that and will ask you to review it. Thanks.